mautic/mautic
# fix: dynamic content timeline entry

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes #12900



#### Description:
Fixes a bug in the contact timeline when an empty dynamic content event appears. The PHP template ([in 4.4](https://github.com/mautic/mautic/blob/4.4/app/bundles/DynamicContentBundle/Views/SubscribedEvents/Timeline/index.html.php)) handled that case correctly, while the twig template seems to have small bug in the condition.

#### Steps to test this PR:
See #12900 to reproduce.

Step 1:
Create a Dynamic Content with a filter, e.g. firstname is not empty. Display on a URL.

Step 2:
Have a contact visit the URL triggering the Dynamic Content.

Step 3:
Go to contact details and check that you don't get any errors, and the dynamic content entry is shown.
